.github/workflows/release_finish.yaml

```diff
@@ -0,0 +1,122 @@
+---
+name: Release Finalize
+on:
+  push:
+    branches:
+      - main
+    paths:
+      # Only trigger for new releases
+      - "version.bzl"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release:
+    if: ${{ github.repository_owner == 'bazelbuild' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          # Get current release version
+          git fetch origin &> /dev/null
+          RELEASE_VERSION=$(cat version.bzl | grep VERSION | sed 's/VERSION = "//' | sed 's/"//')
+          echo "RELEASE_VERSION=${RELEASE_VERSION}" >> $GITHUB_ENV
+
+          # Get release candidate number
+          echo "RELEASE_CANDIDATE_NUMBER=$(git tag -l | grep "${RELEASE_VERSION}" | wc -l | xargs || true)" >> $GITHUB_ENV
+      - run: |
+          # Ensure there is at least 1 release candidate in the environment
+          if [[ -z "${RELEASE_CANDIDATE_NUMBER}" ]]; then
+            exit 1
+          elif [[ "${RELEASE_CANDIDATE_NUMBER}" -eq "0" ]]; then
+            exit 1
+          fi
+      - run: |
+          # Download all artifacts from the release candidate
+          TARGETS=(
+            aarch64-apple-darwin
+            aarch64-unknown-linux-gnu
+            x86_64-apple-darwin
+            x86_64-pc-windows-gnu
+            x86_64-unknown-linux-gnu
+          )
+
+          for triple in ${TARGETS[@]}; do
+            if [[ "${triple}" == *"windows"* ]]; then
+                ext=".exe"
+            else
+                ext=""
+            fi
+            resolver="${ARTIFACT_DIR}/${triple}/release/crate_universe_resolver${ext}"
+            mkdir -p "$(dirname "${resolver}")"
+            url="${ARTIFACT_URL}/crate_universe_resolver-${triple}${ext}"
+            echo "Downloading '${url}' to '${resolver}'"
+            curl --fail -Lo "${resolver}" "${url}"
+            sha256="$(shasum --algorithm 256 "${resolver}" | awk '{ print $1 }')"
+
+            if [[ -z "$(grep ${sha256} ${{ github.workspace }}/crate_universe/private/defaults.bzl)" ]]; then
```

Comment:
Can be a follow-up: Can you make this grep for `"${triple}": "${sha256}"` rather than just the `sha256`?

Reply:
Updated